src/isIsogram.test.js

```diff
@@ -7,13 +7,33 @@ describe(`Function 'isIsogram':`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
 
-  it(`should return a boolean`, () => {
+  it(`should return a 'boolean'`, () => {
+    expect(typeof isIsogram(''))
+      .toBe('boolean');
+  });
 
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('MATE'))
+      .toBe(true);
   });
```

Comment:
You don't check case sensitivity in this test, you should check what happens if the same letter is uppercased and lowercase, like `'Bob'`